modules/administration/nav-administration-guide.adoc

```diff
@@ -32,6 +32,7 @@ endif::[]
 *** xref:custom-channels.adoc[Custom Channels]
 
 ifeval::[{suma-content} == true]
+*** xref:third-party-channels.adoc[Third-Party Channels]
```

Comment:
Why only in "suma-content". 
I think Uyuni user have the same problems

Reply:
Thanks, changed.